warehouse/oso_sqlmesh/oso_metrics/blockchain/layer2_gas_fees.sql

```diff
@@ -3,7 +3,7 @@ select
     events.event_source,
     events.to_artifact_id,
     '' as from_artifact_id,
-    @metric_name('layer2_gas_fees') as metric,
+    @metric_name() as metric,
```

Comment:
```suggestion
    @metric_name('layer2_gas__fees_no_amortization') as metric,
```

This is what actually would fix this